packages/tenant-readmodel-writer/src/index.ts

```diff
@@ -5,16 +5,23 @@ import {
   readModelWriterConfig,
   tenantTopicConfig,
   decodeKafkaMessage,
+  ReadModelRepository,
 } from "pagopa-interop-commons";
 import { runConsumer } from "kafka-iam-auth";
-import { TenantEventV1 } from "pagopa-interop-models";
-import { handleMessage } from "./tenantConsumerService.js";
+import { TenantEvent } from "pagopa-interop-models";
+import { match } from "ts-pattern";
+import { handleMessageV2 } from "./tenantConsumerServiceV2.js";
+import { handleMessageV1 } from "./tenantConsumerServiceV1.js";
+
+const config = readModelWriterConfig();
+const { tenantTopic } = tenantTopicConfig();
+const { tenants } = ReadModelRepository.init(config);
```

Comment:
config handling should be updated to [new approach](https://github.com/pagopa/interop-be-monorepo/blob/main/packages/agreement-readmodel-writer/src/config/config.ts#L15)